services/rook-ceph-cluster/1.10.3/objectbucketclaims/helmrelease.yaml

```diff
@@ -0,0 +1,33 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: object-bucket-claims
+  namespace: ${releaseNamespace}
+spec:
+  chart:
+    spec:
+      chart: object-bucket-claim
+      sourceRef:
+        kind: HelmRepository
+        name: mesosphere.github.io-charts-stable
+        namespace: kommander-flux
+      version: 0.1.0
+  interval: 15s
+  install:
+    crds: CreateReplace
+    remediation:
+      retries: 30
+    createNamespace: true
+  upgrade:
+    crds: CreateReplace
+    remediation:
+      retries: 30
+  releaseName: object-bucket-claims
+  valuesFrom:
+    - kind: ConfigMap
+      name: rook-ceph-cluster-1.10.3-d2iq-defaults
+    - kind: ConfigMap
+      name: rook-ceph-cluster-overrides
```

Comment:
would users be able to configure the OBC with cluster-scoped configuration? (I don't think so unless we can hardcode another configmap name here, right?)

Reply:
hmm.. I guess we should add it here then 😅 it'll be called `rook-ceph-cluster-cluster-overrides` will have to just deal with that stutter :p

Reply:
good catch, add another optional entry.